tools/SetupFlow/DevHome.SetupFlow/Strings/en-us/Resources.resw

```diff
@@ -1919,6 +1919,14 @@
     <value>Generate project</value>
     <comment>Tooltip for the generate button on the QuickstartPlayground page</comment>
   </data>
+  <data name="QuickstartPlaygroundSubmitButton.Content" xml:space="preserve">
+    <value>Submit</value>

```

Comment:
[Don't use submit](https://uxmovement.com/forms/why-your-form-buttons-should-never-say-submit/)  Change the name to the action that will happen when the button is clicked.

Reply:
Same for "Submit prompt"